wildfly-users-ejb/src/main/java/gallb/wildfly/users/ejb/publicationManagement/business/BorrowManager.java

```diff
@@ -0,0 +1,129 @@
+/**
+ * 
+ */
+package gallb.wildfly.users.ejb.publicationManagement.business;
+
+import java.util.Date;
+import java.util.List;
+
+import javax.ejb.Stateless;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.persistence.PersistenceException;
+
+import org.jboss.logging.Logger;
+
+import gallb.wildfly.users.common.IBorrow;
+import gallb.wildfly.users.common.IEntity;
+import gallb.wildfly.users.ejb.exception.EjbException;
+import model.Borrow;
+import model.Publication;
+import model.User;
+
+/**
+ * @author nagys, gallb
+ *
+ */
+@Stateless
+public class BorrowManager implements IBorrow {
+
+	@PersistenceContext(unitName = "WildflyUsers")
+	private EntityManager oEntityManager;

```

Comment:
Nem ertem, ez egy business vagy DAO osztaly? Mert van benne business logic es az adatbazissal is direktbe kommunikal.